virt_lightning/__init__.py

```diff
@@ -263,7 +276,15 @@ def root_password(self, root_password=None):
         if chpassd:
             return chpassd["list"].split(":")[1]
 
-    def ssh_key_file(self, ssh_key_file):
+    @property
+    def ssh_key(self):
+        return self._ssh_key
+
+    @ssh_key.setter
+    def ssh_key(self, value):
```

Comment:
In next task need validate ssh_key value for correct key. (not required for this review)

Reply:
Good idea! Could you open an issue to trace the idea.